README.md

```diff
@@ -1,8 +1,11 @@
 # ToxiproxyEx
 
-[Online Documentation](https://hexdocs.pm/toxiproxy_ex).
+[![hex.pm](https://img.shields.io/hexpm/v/toxiproxy_ex.svg)](https://hex.pm/packages/toxiproxy_ex)
```

Comment:
Could you share some description about each badge so that we can decide which ones to keep?

I'm not a big fan of having all those badges unless they serve a good purpose.

Reply:
Badges are mostly for a quick glance or overview of a module.

[![hex.pm](https://img.shields.io/hexpm/v/toxiproxy_ex.svg)](https://hex.pm/packages/toxiproxy_ex)
The latest module version.

[![hexdocs.pm](https://img.shields.io/badge/api-docs-lightgreen.svg)](https://hexdocs.pm/toxiproxy_ex/api-reference.html)
Link directly to the API list documentation.

[![hex.pm](https://img.shields.io/hexpm/dt/toxiproxy_ex.svg)](https://hex.pm/packages/toxiproxy_ex)
How often this module was used by others.

[![hex.pm](https://img.shields.io/hexpm/l/toxiproxy_ex.svg)](https://hex.pm/packages/toxiproxy_ex)
What license being used. Sometimes there is a mismatch between the license in `mix.exs` and actual `LICENSE` file.

[![github.com](https://img.shields.io/github/last-commit/Jcambass/toxiproxy_ex.svg)](https://github.com/Jcambass/toxiproxy_ex/commits/master)
When was this module was updated? It's still actively developed or maintained?

We miss out the GitHub's CI result badges.

In your call. :smile:

Reply:
Right :) I guess you're right in the sense that all of them provide some value. Let's add them  🎉

Reply:
Maybe you could also add this one for CI:

![.github/workflows/main.yml](https://github.com/Jcambass/toxiproxy_ex/workflows/.github/workflows/main.yml/badge.svg?branch=main)

Reply:
The latest commit one would need to point to `main` instead of `master`.